nfs/src/rpc_stats.cpp

```diff
@@ -174,6 +174,7 @@ do { \
     DUMP_OP(FUSE_ACCESS);
     DUMP_OP(FUSE_GETATTR);
     DUMP_OP(FUSE_SETATTR);
+    DUMP_OP(FUSE_STATFS);
```

Comment:
nit, this is best kept as the first argument as it pertains to the entire filesystem while all others pertain to individual files.